mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/263 - confirmation dialog bug

Closes #263 

To test:
- Run Training/Stop Worms scenario
- Play through scenario until the end
- Make sure confirmation dialogs accept answers you give them and pause the scenario while up

Note:
- Server process seems to just exit when scenario is done, I think we're not catching the server event for successfully completing a scenario
